src/routes/pay/stripe/index.ts

```diff
@@ -18,10 +18,6 @@ import { createRefundTxs, updateTxState } from './transaction'
 
 const logger = getLogger('route-stripe')
 
-const stripe = new Stripe(environment.stripeSecret, {
-  apiVersion: '2020-08-27',
```

Comment:
API version will be update-to-date with `stripe` package if we don't specify a `apiVersion`?

Reply:
start from v12,it  seem have to specify a apiVersion